src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -1663,9 +1663,6 @@
   <data name="ArgumentNull_AssemblyName" xml:space="preserve">
     <value>AssemblyName cannot be null.</value>
   </data>
-  <data name="ArgumentNull_AssemblyNameName" xml:space="preserve">
```

Comment:
Is this being referenced from the native side?
https://github.com/dotnet/runtime/blob/7bd46666716a0c7311b0e28594238fe4fa2d3b60/src/coreclr/vm/assembly.cpp#L397

Reply:
Yup